packages/communication-react/package.json

```diff
@@ -64,6 +64,9 @@
   },
   "main": "./dist/dist-cjs/communication-react/index.js",
   "module": "./dist/dist-esm/communication-react/src/index.js",
+  "exports": {
```

Comment:
Can you double check if we should also include a default `"."` route ? I see most examples do include it but nothing definitive in the docs, I assume things still check the "main" field but it might be safer to include anyways?

Reply:
yeah if most examples are showing this we might be missing something, I will add it and test like we did today and then update if need be